subsys/net/ip/ipv6.c

```diff
@@ -623,17 +623,9 @@ enum net_verdict net_ipv6_process_pkt(struct net_pkt *pkt, bool is_loopback)
 	case IPPROTO_ICMPV6:
```

Comment:
Commit messages typos:

> As for IPv4, net_conn_input() can be called at **once** place for udp/tcp.
It will anyway check if **gived** protocol is enabled so no need to check
such support here.